app/assets/stylesheets/application.css.scss

```diff
@@ -62,23 +69,16 @@ input {
   border: 0px;
 }
 
-
-.navbar-form > form > input[type="text"] {
- border-radius: 4px;
- border: 0px;
- width: 200px;
- height: 30px;
-}
-
 .navbar-default .navbar-nav > li > a {
  color: white; /*Change active text color here*/
 }
 
 .navbar-default .navbar-brand {
- color: #9ECAE1; /*Change active text color here*/
+ color: #9ECAE1;
```

Comment:
Color literals like `#9ECAE1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#9ECAE1` should be written as `#9ecae1`<br>Line should be indented 2 spaces, but was indented 1 spaces